src/ui/components/VariantSelector.tsx

```diff
@@ -56,7 +56,7 @@ export function VariantSelector(props: { variants: { id: string; name: string }[
 									searchParams.get("variant") === variant.id
 										? "border-transparent bg-gray-900 text-white hover:bg-gray-800"
 										: "border-gray-200 bg-white text-gray-900 hover:bg-gray-100",
-									"flex items-center justify-center overflow-hidden text-ellipsis rounded border p-3 text-center text-sm font-semibold focus-within:outline focus-within:outline-2",
+									"flex min-w-[8ch] items-center justify-center overflow-hidden text-ellipsis whitespace-nowrap rounded border p-3 text-center text-sm font-semibold focus-within:outline focus-within:outline-2",
```

Comment:
```suggestion
									"flex min-w-max items-center justify-center overflow-hidden text-ellipsis whitespace-nowrap rounded border p-3 text-center text-sm font-semibold focus-within:outline focus-within:outline-2",
```
Wouldn't that be better?

Reply:
I set it to this fixed value because of labels like 's', 'm', 'l', etc. By setting a minimal fixed width to 8 chars in this case, it looks better in my opinion.
![11](https://github.com/saleor/storefront/assets/27455716/2cad55d0-caa7-4f10-809b-2057d60f7cfb)